blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Version v0.22 release"
+linkTitle: "Version v0.22 release"
```

Comment:
Do we need "Version" and "v" in the title?

Reply:
I will remove the small `v`